workflow-language/src/main/resources/swadl-schema-1.0.json

```diff
@@ -666,7 +666,8 @@
             },
             "required": [
                 "message-received"
-            ]
+            ],
+            "additionalProperties": true
```

Comment:
do we have to change all events?

Reply:
It looks like this is not needed since we don't consider timeout if defined inside the event. Adding it for "one-of" property is enough. I'll remove it in next commit.